projects/Grandine-Support-Documentation.md

```diff
@@ -0,0 +1,79 @@
+# Grandine Support and Documentation
+
+This project is focused on building out a more comprehensive support and documentation book for Grandine through research and trial and error. 
+
+## Motivation
+
+Grandine is the newest Consensus Layer client and the documentation is on the lighter side. I think it's important to have more comprehsneive documentation around Grandine and its capabilities. I also think this will help attract new users to the client, which would increase client diversity among CL clients.
+
+## Project description
+
+I am proposing to add more support and documentation to [Grandine's book](https://github.com/grandinetech/grandine/tree/develop/book) which will require me to become intimately familiar with Grandine through research and trial and error. 
+
+The following areas are ones I think should be addressed but this is likely to change as I begin making my way deeper into Grandine. I am taking inspiration from the Lighthouse and Prysm documentation books, so whilst not everything that they have will be applicable to Grandine, I think it is a good base to work from.
+- ### Installation
```

Comment:
what operating systems are you focusing on? Or are you going to focus on docker images specifically?

Reply:
@Karrenbelt I'm using MacOS. I know some other fellows are using Windows though, ultimately would like to get all operating systems documented. :)